src/retriever.js

```diff
@@ -7,9 +7,8 @@ var url = require('url');
 
 function Retriever(options) {
   options = options || {};
-  this._host = options.host || 'pixabay.com';
-  this._path = options.path || 'api';
-  this._protocol = options.protocol || 'https';
+
+  this._path = options.url || 'https://pixabay.com/api';
```

Comment:
I know I'm really harping on naming, but can we just call this a URL all the way through?